### PR TITLE
Update SQA documentation

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #127 )

### CODE REVIEW

**Review by bwspenc — CHANGES_REQUESTED** [review]
Looks good. Just a couple of questions. The only change needed is to refer to a real issue.

**Comment by bwspenc** [comment]
Looks like #127 would work for an issue.

File: doc/config.yml [hunk 1]
```diff
@@ -23,7 +24,7 @@ Extensions:
         Input Structure: getting_started/BlackBearInputStructure.md
       Code Reference:
         Systems: syntax/index.md
-        Software Quality: sqa/blackbear_index.md
```
[the remark this answers is not in the document]
Comment (aeslaughter) [reply doc/config.yml:26]: Just using the template that contains all the same information.

File: doc/content/sqa/blackbear_far.md [hunk 2]
```diff
@@ -0,0 +1 @@
+!template load file=app_far.md.template app=BlackBear category=blackbear
```
[the remark this answers is not in the document]
Comment (aeslaughter) [reply doc/content/sqa/blackbear_far.md:1]: Yes. https://mooseframework.inl.gov/blackbear/docs/PRs/181/site/sqa/

**Comment by aeslaughter** [comment]
I updated the issue in the title

**Pull request merged**